graphics/src/test/kotlin/ktx/graphics/graphicsTest.kt

```diff
@@ -106,6 +106,28 @@ class GraphicsTest {
     verify(batch).end()
   }
 
+  @Test
+  fun `should begin with provided projection matrix`() {
+    val batch = mock<Batch>()
+    val matrix = Matrix4(FloatArray(16) {it.toFloat()})
+
+    batch.begin(projectionMatrix = matrix)
```

Comment:
I guess it comes down to preference, but I generally tend to visually separate setup phase, tested method and assets. I'd add a newline here.

Reply:
Same goes for the other test.

Reply:
Done